plugins/SegmentEditor/SegmentSelectorControl.php

```diff
@@ -94,9 +94,7 @@ public function getClientSideProperties()
 
     private function wouldApplySegment($savedSegment)
     {
-        $isBrowserArchivingDisabled = Config::getInstance()->General['browser_archiving_disabled_enforce'];
-
-        if (!$isBrowserArchivingDisabled) {
+        if (!$this->isBrowserArchivingDisabledForSegments()) {
```

Comment:
Maybe `ArchiveProcessor\Rules::isBrowserArchivingAvailableForSegments()` could be used instead